rootfs/etc/profile.d/_07-term-mode.sh

```
@@ -149,12 +208,12 @@ function _srgb_to_luminance() {
 	}
 
 	# Linearize each color component
-	local R=$(normalize_and_linearize $red)
-	local G=$(normalize_and_linearize $green)
-	local B=$(normalize_and_linearize $blue)
+	R=$(normalize_and_linearize $red)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)